src/code/z_actor.c

```diff
@@ -4445,39 +4445,42 @@ s32 func_800BE63C(struct EnBox* chest) {
     return false;
 }
 
-TexturePtr D_801AEFA8[] = {
-    gameplay_keep_Tex_091DE0,
-    gameplay_keep_Tex_091FE0,
-    gameplay_keep_Tex_0921E0,
-    gameplay_keep_Tex_0923E0,
+TexturePtr sElectricShockTextures[] = {
+    gElectricShock1Tex,
+    gElectricShock2Tex,
+    gElectricShock3Tex,
+    gElectricShock4Tex,
 };
 
-// Draw common damageEffects
-void func_800BE680(GlobalContext* globalCtx, Actor* actor, Vec3f limbPos[], s16 arg3, f32 effectScale, f32 steamScale,
-                   f32 effectAlpha, u8 mode) {
+/**
+ * Draw common damage effects applied to each limb provided in limbPos
+ */
+void Actor_DrawDamageEffects(GlobalContext* globalCtx, Actor* actor, Vec3f limbPos[], s16 limbPosCount, f32 effectScale,
+                             f32 frozenSteamScale, f32 effectAlpha, u8 type) {
     if (effectAlpha > 0.001f) {
```

Comment:
Does this still match if it were an early return? I think it would make things more readable